distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,79 @@
+priority "grant";
+
+deny {
```

Comment:
❓ Is this deny block necessary? Since you're using the double-equals notation when setting the -Djava.security.policy, aren't these permissions are denied unless explicitly granted?

Reply:
This is how the pro-grade security manager works and because I set the priority to grant to avoid additional denied permissions.